src/themes/OLH/assets/js/table_of_contents.js

```diff
@@ -12,7 +12,8 @@ $( document ).ready(function() {
     $("#main_article h2, #main_article h3").each(function () {
 
         link = $(this);
-        title = link.text();
+        link.find('a').contents().unwrap();
+        title = link.html();
```

Comment:
Does this solve the problem? If I'm reading this correctly and following the example from Patrick, we are unwrapping (destroying) the anchor tag, but leaving in the <sup> tag inside the HTML?

I don't see any code removing the <sup> tag, so I think it will still be rendered, (though as an actual <sup> rather than as text)

Reply:
The thing that was breaking the TOC was the <a> tag inside the <sup>.

At this point we have conflicting requests. Glossa want the title in the TOC represented exactly as it is in the text that includes the <sup> so I chose displaying the <sup> properly.

Reply:
Where is the request from Glossa?

Reply:
> Where is the request from Glossa?

Via email. Personally I think the TOC should display the exact title even if it includes a `<sup>`.

Reply:
> > Where is the request from Glossa?
> 
> Via email. Personally I think the TOC should display the exact title even if it includes a `<sup>`.

I think it is worth running it past both folks again. If we close this as resolved we won't actually resolve the original request

Reply:
I agree a footnote number might look a little odd in the TOC but I'm OK with removing that as a separate request. The light touch on this PR is OK for now